pkg/operator/ceph/nfs/nfs.go

```diff
@@ -37,6 +38,8 @@ import (
 
 const (
 	ganeshaRadosGraceCmd = "ganesha-rados-grace"
+	dotNFSPoolName       = ".nfs"
```

Comment:
We need a more meaningful name for this. Like `defaultNFSPoolName`? Also add a comment this is hardcoded in Ceph.

Reply:
[here](https://github.com/rook/rook/pull/8501/files#diff-25dee7641dd209b1bb92d0c8aae2c496d9d282d4af7554ff8b5f6125bfa11831R42)